### PR TITLE
env sample updated for VAPID key

### LINKED ISSUE #55: Missing FCM_VAPID_KEY in sample .env file
.env file missing a REACT_APP_FIREBASE_FCM_VAPID_KEY variable. Throws the following error.

Screenshot 2021-04-10 at 12 18 57

### PR DESCRIPTION
**Pull request by soham4abc**
Layout for vapid key added in the env.sample

## Motivation and Context
This change helps new contributors to set up the project with ease
fix #55 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -2,4 +2,5 @@
  REACT_APP_FIREBASE_PROJECT_ID=<FIREBASE_PROJECT_ID>
  REACT_APP_FIREBASE_MESSAGING_SENDER_ID=<FIREBASE_MESSAGING_SENDER_ID>
  REACT_APP_FIREBASE_APP_ID=<FIREBASE_APP_ID>
- REACT_APP_FIREBASE_MEASUREMENTID=<FIREBASE_MEASUREMENTID>
\ No newline at end of file
+ REACT_APP_FIREBASE_MEASUREMENTID=<FIREBASE_MEASUREMENTID>
+ REACT_APP_FIREBASE_FCM_VAPID_KEY<VAPID_KEY>
```
Comment (thisuraseniya) [inline .env.sample:6]: Missing equals sign

**Review by thisuraseniya — COMMENTED** [review]
equals sign ( = ) is missing

File: .env.sample (hunk 1 above)
Comment (soham4abc) [reply .env.sample]: Resolved. Please have a look!

**Comment by soham4abc** [comment]
@thisuraseniya  Please have a look

**Comment by thisuraseniya** [comment]
This change is already updated by another PR merged previously. So closing this.

**Pull request closed without merging**